tests/time_tests/test_runner/test_timetest.py

```diff
@@ -63,7 +63,7 @@ def test_timetest(instance, executable, niter, cl_cache_dir, test_info, temp_dir
         run_timetest(_exe_args, log=logging)
         assert os.listdir(cl_cache_dir), "cl_cache isn't generated"
 
-    retcode, aggr_stats = run_timetest(exe_args, log=logging)
+    retcode, aggr_stats, _ = run_timetest(exe_args, log=logging)
```

Comment:
AFAIK, we are interested in saving raw results in DB to have historical data for any experiments. So, need to add `test_info["raw_results"] = raw_results` below (on 71 line of the file)